consensus/reactor.go

```diff
@@ -814,6 +819,16 @@ func (conR *ConsensusReactor) StringIndented(indent string) string {
 	return s
 }
 
+func (conR *ConsensusReactor) setCatchingUp() {
+	var catchingUp float64
+	if conR.fastSync {
```

Comment:
"Whether or not a node is synced. 0 if syncing, 1 if synced." **1 and 0 mixed up**

if you're adding this condition, you need to modify metric name and description to be

`"Whether or not a node is fast synced. 1 if fast syncing, 0 if synced or syncing without fast sync."`

Reply:
Ah, nice catch.